api-lib/findUser.ts

```diff
@@ -72,3 +72,34 @@ export const getUserFromAddress = async (address: string, circleId: number) => {
 
   return users.pop();
 };
+
+export const getUsersFromUserIds = async (
+  userIds: number[],
+  circleId: number
+) => {
+  const { users } = await adminClient.query({
+    users: [
+      {
+        where: {
+          id: { _in: userIds },
+          circle_id: { _eq: circleId },
```

Comment:
nit: the circle_id shouldn't be needed, since users are all tied to a single circle. I don't think adding it should even optimize the query at all, given that the index still needs to be traversed to resolve the user ids regardless.

Reply:
I added the `circle_id` as a check to validate the `teammate_ids` sent by the client are indeed same circle as the user and so invalid users (from other circles) could be filtered out.